### PR TITLE
Update Example Links

### PR DESCRIPTION
**Pull request by rohitjmathew**
Issue #1246 didn't have anyone assigned to it so I took it up to add a few examples but as @KKVANONYMOUS started the original issue I have also left a few examples out so he can also have a shot at contributing to hacktoberfest.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -114,6 +114,10 @@ Here are some resources to get you started with Earthly
   * [C++](./examples/cpp)
   * [Mono-repo](./examples/monorepo)
   * [Multi-repo](./examples/multirepo)
+  * [Python](./examples/python)
+  * [Ruby](./examples/ruby)
+  * [Rust](./examples/rust)
+  * [Terraform](./examples/terraform)
```
Comment (vladaionescu) [inline README.md:120]: The terraform example isn't very reliable. I think it's better not to feature it.

Also, the examples should be ordered better - e.g. the language examples should sit together.

**Review by vladaionescu — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (rohitjmathew) [reply README.md]: I have sorted them alphabetically as per @alexcb comment [here](https://github.com/earthly/earthly/pull/1253/files#r721852823). I have also removed the terraform example like you mentioned.

**Comment by alexcb** [comment]
This has been done in https://github.com/earthly/earthly/pull/1253

**Comment by rohitjmathew** [comment]
@alexcb I see you manually added the missing languages in this [commit](https://github.com/earthly/earthly/commit/b29d7222aafc916586bd499e2abb701e386a2555). No worries. Thanks for the opportunity to contribute 😄

**Pull request closed without merging**